helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -285,6 +294,14 @@ private Response batchGetStoppableInstances(String clusterId, JsonNode node, boo
         }
       }
 
+      if (selectionBase != InstanceHealthSelectionBase.non_topo_based && clusterTopology.getZones()
```

Comment:
Why do we need the user to explicitly provide a sectionBase? Can't we simplify this API by figuring out the InstanceHealthSelectionBase internally?

if orderOfZone is provided we use zone_based
if necessary topology configs are set, we use cross_zone_based
else we use non_topo_based

Reply:
This change may potentially break backward compatibility for users who intend to check only a single zone but didn't provide an orderOfZone.

Reply:
Discussed offline. Will make NON_TOPO be the default value if the selection base is not provided. In a separate PR will support automatically detect whether to use NON_TOPO_BASE or CROSS_ZONE_BASE if the user didn't specify the selection base.